cymetric/convenient_interface.py

```diff
@@ -0,0 +1,121 @@
+from pyne import nucname
+import pandas as pd
+import numpy as np
+import cymetric as cym
+
+def GetTransactionTimeSeries(db, sender='All', receiver='All', *xargs):
+
+  
+  nuc_list = []
+  for inx, nuc in enumerate(xargs):
+    nuc_list.append(nucname.id(nuc))
+
+  #initiate evaluation
+  evaler = cym.Evaluator(db)
+  
+  # get transation & Agent tables
+  trans = evaler.eval('Transactions')
+  agents = evaler.eval('AgentEntry')
+  
+# build 2 table for SenderId and ReceiverId
+  # get receiver
+  agents_receiver = agents.rename(index=str, columns={'AgentId': 'ReceiverId'})
+  if receiver != 'All':
+    agents_receiver_ = agents_receiver.loc[lambda df: df.Prototype == receiver,:]
+    # check if receiver exists
+    if agents_receiver_.empty:
```

Comment:
I get it from:
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.empty.html